src/Environments/MacOS/Classic/ResourceFork.cs

```diff
@@ -494,6 +497,79 @@ private int GetRunLengthValue(BeImageReader a5dr, ref int a5repeat)
             return runLength;
         }
 
+        private void A5Relocate(BeImageReader a5dr, UInt32 a5dbelow)
+
+        // Update pointers within A5World that point to an address within A5World, to the real/absolute address. 
```

Comment:
Please move the comment to before the `private void A5Relocate(...`  and format it consistently with other comments in the code:
```C#
/// <summary>
/// Update pointers within A5 World that point to an address within A5World, to the real/absolute address. 
/// </summary>
/// <param name="a5rdr">An image reader to the segment containing the A5 World,</param>
/// <param name="a5dbelow">Offset to the location of the globals in the A5 world.</param>
```